mlflow/R/mlflow/R/model-serve.R

```diff
@@ -4,8 +4,7 @@
 #'
 #' Serves an RFunc MLflow model as a local web API.
 #'
-#' @param model_path The path to the MLflow model, as a string.
-#' @param run_id ID of run to grab the model from.
+#' @param model_uri The URI to the MLflow model, as a string.
```

Comment:
I don't think we have to copy the list of URI examples for every docstring, but I think it's worth at least putting 
```
#                  For information about supported URI schemes, see the
#                  `Artifacts Documentation
#                  <https://www.mlflow.org/docs/latest/tracking.html#supported-artifact-stores>`_.
```

Reply:
Good point, although it's a bit verbose to include everywhere. Would it make sense to just say <mlflow-artifact-uri> and expect users are able to search the docs?

I am ok copying that here as well, just in general I don't like if there is too much text in the comment.

Reply:
I don't think it'd hurt to include the information everywhere (preferably in the R doc so people won't have to follow a link). Also since `model_uri` has the same doc across functions we can consider [inheriting param](http://r-pkgs.had.co.nz/man.html) docs or using a template (e.g. https://github.com/mlflow/mlflow/blob/0cd28559447fd3a5d993aca66263060373183493/mlflow/R/mlflow/R/tracking-experiments.R#L8)

Reply:
I've updated these to use the template argument, thanks for the suggestion!